src/app/services/bigquery.service.ts

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
-import { environment } from '../../environments/environment';
-import { MAX_RESULTS, TIMEOUT_MS } from '../app.constants';
+import {environment} from '../../environments/environment';
+import {MAX_RESULTS, TIMEOUT_MS} from '../app.constants';
```

Comment:
Same here. Perhaps Angular linter and the one managed by your editor aren't aligned?